src/views/Login/Login.vue

```diff
@@ -50,18 +69,54 @@ export default {
   name: 'Login',
   data() {
     return {
-      username: '',
-      password: ''
+      errorMsg: {
+        title: null,
+        action: null
+      },
+      userInfo: {
+        username: null,
+        password: null
+      },
+      disableSubmitButton: false,
+      submitButtonText: 'Login'
     };
   },
   methods: {
+    validateForm: function() {
+      this.errorMsg.title = null;
+      this.errorMsg.action = null;
+
+      // Name error message
```

Comment:
update this comment to be more correct? Or remove?

Reply:
Removed in new commit.